### PR TITLE
Update capabilities for 1.0 Rev3

### PR DESCRIPTION
**Pull request by dneto0**

### CODE REVIEW

**Comment by dneto0** [comment]
Rebased against master

File: source/operand.cpp [hunk 1]
```diff
@@ -1011,102 +1011,73 @@ static const spv_operand_desc_t capabilityInfoEntries[] = {
...
 // If the source arrays were std::array, then we could have used
 // std::array::size.
-#define ARRAY_SIZE(A) (static_cast<uint32_t>(sizeof(A)/sizeof(A[0])))
+#define ARRAY_SIZE(A) (static_cast<uint32_t>(sizeof(A) / sizeof(A[0])))
 
 static const spv_operand_desc_group_t opcodeEntryTypes[] = {
-  // TODO(dneto): Reformat this table.
-    {SPV_OPERAND_TYPE_SOURCE_LANGUAGE,
-     ARRAY_SIZE(sourceLanguageEntries),
+    // TODO(dneto): Reformat this table.
```
Comment (dekimir) [inline source/operand.cpp]: Is this now done?

File: test/Validate.Capability.cpp [hunk 2]
```diff
@@ -514,43 +516,26 @@ make_pair("OpDecorate %intt BuiltIn PatchVertices\n",             TessellationDe
...
 make_pair("OpDecorate %intt BuiltIn VertexIndex\n",               ShaderDependencies()),
 make_pair("OpDecorate %intt BuiltIn InstanceIndex\n",             ShaderDependencies()),
-// Though the remaining builtins don't require Shader, the BuiltIn keyword
-// itself currently does require it.
-make_pair("OpDecorate %intt BuiltIn NumWorkgroups\n",             ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn WorkgroupSize\n",             ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn WorkgroupId\n",               ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn LocalInvocationId\n",         ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn GlobalInvocationId\n",        ShaderDependencies()),
-make_pair("OpDecorate %intt BuiltIn LocalInvocationIndex",        ShaderDependencies()),
```
Comment (dekimir) [inline test/Validate.Capability.cpp]: The above six cases disappeared, but I would have expected them to stay with `AllDependencies()` -- what am I missing?

**Comment by dekimir** [comment]
-1 for deleted tests (though I could be convinced that's OK after a discussion).

File: test/Validate.Capability.cpp (hunk 2 above)
Comment (dneto0) [inline test/Validate.Capability.cpp]: I had the sense backward.  You're right, we should test those with AllCapabilities().

**Comment by dneto0** [comment]
All feedback addressed:
- re-added tests for BuiltIns without a capability dependency
- fix LocalInvocationIndex: It does not depend on a capability.

**Comment by dekimir** [comment]
+2

**Comment by dneto0** [comment]
Squashed and pushed into master as  dd8a661b

**Pull request closed without merging**